ocs_ci/utility/deployment_openshift_logging.py

```
@@ -605,22 +619,17 @@ def install_logging(skip_resource_exists=False):
-        raise Exception("Failed to get configmap")
-
+    # Wait for ConfigMap to be created by OBC provisioner
     configmap_obj = ocp.OCP(
         kind=constants.CONFIGMAP, namespace=constants.OPENSHIFT_LOGGING_NAMESPACE
     )
+    if not configmap_obj.check_resource_existence(
+        resource_name=constants.OBJECT_BUCKET_CLAIM,
+        should_exist=True,
+        timeout=180,
+    ):
+        raise Exception("Failed to get configmap")
```

Ruff (0.16.1)

[warning] Create your own exception

(TRY002)